api/src/test/java/io/strimzi/api/kafka/model/kafka/KafkaCrdIT.java

```diff
@@ -74,8 +74,8 @@ public void testKafkaWithNullMaintenance() {
                 () -> createDeleteCustomResource("Kafka-with-null-maintenance.yaml"));
 
         assertThat(exception.getMessage(), anyOf(
-                        containsStringIgnoringCase("invalid: spec.maintenanceTimeWindows: Invalid value: \"null\": spec.maintenanceTimeWindows in body must be of type string: \"null\""),
-                        containsStringIgnoringCase("invalid: spec.maintenanceTimeWindows[0]: Invalid value: \"null\": spec.maintenanceTimeWindows[0] in body must be of type string: \"null\"")));
+                        containsStringIgnoringCase("spec.maintenanceTimeWindows: Invalid value: \"null\": spec.maintenanceTimeWindows in body must be of type string: \"null\""),
+                        containsStringIgnoringCase("spec.maintenanceTimeWindows[0]: Invalid value: \"null\": spec.maintenanceTimeWindows[0] in body must be of type string: \"null\"")));
```

Comment:
how this is related to the metrics reporter?

Reply:
https://github.com/strimzi/strimzi-kafka-operator/pull/11051#pullrequestreview-2639985246